numba_dpex/_patches.py

```diff
@@ -230,22 +304,28 @@ def _empty_nd_impl(context, builder, arrtype, shapes):
         )
 
     if isinstance(arrtype, DpnpNdArray):
+        (queue_ref, queue_ptr, pyapi) = make_queue(context, builder, arrtype)
+        # This might fix the segfault
+        # sycl_queue_val = cgutils.alloca_once(
+        #     builder,
+        #     utils.get_llvm_type(context=context, type=types.voidptr),
```

Comment:
The problem here is that the context here is LVVM context and not Numba CpuContext.